Fusion/src/FusionAhrs.c

```diff
@@ -36,9 +36,9 @@
 void FusionAhrsInitialise(FusionAhrs *const ahrs) {
```

Comment:
What was the reason to split .c and .h files into src and include directories? This way we will not be able to easily benefit from any updates made to the original repo.